RELEASE_NOTES.md

```diff
@@ -1,5 +1,1069 @@
+    token = "<api_key>"
+  }
+}
+```
+
+#### v1.48.0
+
+```
+resource "ibm_cd_toolchain_tool_artifactory" "my_artifactory_tool" {
+  toolchain_id = ibm_cd_toolchain.toolchain.id
+  parameters {
+    name = "artifactory-tool-01"
+    dashboard_url = "https://mycompany.example.jfrog.io"
+    type = "docker"
+    user_id = "<user_id>"
+    repository_name = "default-docker-local"
+    repository_url = "https://mycompany.example.jfrog.io/artifactory/default-docker-local"
+    api_key = "<api_key>"
+  }
+}
+```
+
+### ibm_cd_toolchain_tool_jenkins
+
+A nested property within the `parameters` object is changed:
+
+- `webhook_url` is now a computed property
+
+### ibm_cd_toolchain_tool_keyprotect
+
+Some nested properties within the `parameters` object are renamed:
+
+- `region` --> `location` - Prefix `ibm:yp:` is no longer required on values
+- `resource_group` --> `resource_group_name`
+
+Example:
+
+#### v1.46.0
+
+```
+resource "ibm_cd_toolchain_tool_keyprotect" "my_keyprotect_tool" {
+  toolchain_id = ibm_cd_toolchain.toolchain.id
+  parameters {
+    name = "kp_tool_01"
+    instance_name = "Key Protect-XX"
+    region = "ibm:yp:us-south"
+    resource_group = "Default"
+  }
+}
+```
+
+#### v1.48.0
+
+```
+resource "ibm_cd_toolchain_tool_keyprotect" "my_keyprotect_tool" {
+  toolchain_id = ibm_cd_toolchain.toolchain.id
+  parameters {
+    name = "kp_tool_01"
+    instance_name = "Key Protect-XX"
+    location = "us-south"
+    resource_group_name = "Default"
+  }
+}
+```
+
+### ibm_cd_toolchain_tool_nexus
+
+A nested property within the `parameters` object is renamed:
+
+- `dashboard_url` --> `server_url`
+
+Example:
+
+#### v1.46.0
+
+```
+resource "ibm_cd_toolchain_tool_nexus" "my_nexus_tool" {
+  toolchain_id = ibm_cd_toolchain.toolchain.id
+  parameters {
+    name = "my-nexus"
+    type = "npm"
+    user_id = "<user_id>"
+    token = "<token>"
+    dashboard_url = "https://my.nexus.server.com/"
+  }
+}
+```
+
+#### v1.48.0
+
+```
+resource "ibm_cd_toolchain_tool_nexus" "my_nexus_tool" {
+  toolchain_id = ibm_cd_toolchain.toolchain.id
+  parameters {
+    name = "my-nexus"
+    type = "npm"
+    user_id = "<user_id>"
+    token = "<token>"
+    server_url = "https://my.nexus.server.com/"
+  }
+}
+```
+
+### ibm_cd_toolchain_tool_pagerduty
+
+Creating a new PagerDuty service is no longer supported. The PagerDuty service must already exist. As a result, some nested properties within the `parameters1 object are removed:
+
+- `key_type`
+- `api_key`
+- `service_name`
+- `user_email`
+- `user_phone`
+- `service_url`
+
+Some nested properties within the `parameters` object are changed:
+
+- `service_key` is now a required property
+- `service_url` is now a required property
+
+Example:
+
+#### v1.46.0
+
+```
+resource "ibm_cd_toolchain_tool_pagerduty" "my_pagerduty_toolchain" {
+  toolchain_id = ibm_cd_toolchain.toolchain.id
+  parameters {
+    key_type = “api“
```

Comment:
Small nit, but some of the double quotes came out a bit funny

![image](https://user-images.githubusercontent.com/5765072/201690893-05cd9948-b2bc-44fa-a46c-e0cd0f98b00b.png)